reports.py

```diff
@@ -715,7 +746,11 @@ def plot_coverage(
 
         plt.title(plot_title, fontsize=font_size * 1.2)
         plt.xlabel("bp", fontsize=font_size * 1.1)
-        plt.ylabel("read depth", fontsize=font_size * 1.1)
+        
+        ylabel = "read depth"
+        if(bin_size > 1):
+            ylabel = "read depth (max in " + str(bin_size)+ "-bp bin)"
```

Comment:
Not important, but the `str` casting and concatenation can be avoided via `format()` (maybe a bit more readable):
`ylabel = "read depth (max in {size}-bp bin)".format(size=bin_size)`

Reply:
Sure thing.